src/test/java/pl/arturkufa/pack/HuffmanAlgorithm/HuffmanComparatorTest.java

```diff
@@ -0,0 +1,55 @@
+package pl.arturkufa.pack.HuffmanAlgorithm;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Map;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+
+class HuffmanComparatorTest {
+
+    private final HuffmanComparator<Character,Integer> huffmanComparator = new HuffmanComparator<>();
+
+    @Test
+    void shouldReturnOneWithBiggerInteger() {
+
+        Map.Entry<Character,Integer> entry1 = mock(Map.Entry.class);
```

Comment:
możesz używać adnotacji @Mock. Metoda mock też nie jest zła. Ale używając adnotacji robisz to raz na górze klasy i nie musisz pisać tego per test